### PR TITLE
The PR Taketh Away: Allows tails, horns, and frills to be removed in character customization

### PR DESCRIPTION
**Pull request by Lyroy**
## About The Pull Request

Makes it so your little freaks of nature can be missing their tail. Works on all races.
Same thing for horns and frills, for Sisseans, Drakians, and Kobolds.
Also has the side-effect of allowing kobolds to pick a color for their tail.

## Why It's Good For The Game

Some characters might not have tails. Some might have lost them. Prevents the rest of players from having a delicious meal of lizard tail.

### CODE REVIEW

File: code/modules/client/customizer/customizers/organ/frills.dm [hunk 1]
```diff
@@ -11,6 +12,7 @@
 /datum/customizer/organ/frills/lizard
 	customizer_choices = list(/datum/customizer_choice/organ/frills/lizard)
 
+
```
Comment (Cyprex) [inline code/modules/client/customizer/customizers/organ/frills.dm:15]:
```suggestion
```

File: code/modules/mob/living/carbon/human/species_types/furry/tabaxi.dm [hunk 2]
```diff
@@ -57,7 +57,7 @@
 		ORGAN_SLOT_LIVER = /obj/item/organ/liver,
 		ORGAN_SLOT_STOMACH = /obj/item/organ/stomach,
 		ORGAN_SLOT_APPENDIX = /obj/item/organ/appendix,
-		ORGAN_SLOT_TAIL = /obj/item/organ/tail/cat,
+		//ORGAN_SLOT_TAIL = /obj/item/organ/tail/cat, //Commenting out due to use of customizer organs.
```
Comment (Cyprex) [inline code/modules/mob/living/carbon/human/species_types/furry/tabaxi.dm:60]: Is this actually correct? How do other customizer-using organs handle this?

Comment (Lyroy) [reply code/modules/mob/living/carbon/human/species_types/furry/tabaxi.dm:60]: Yup, check the tiefling code, for example.

**Comment by ikrarkjr** [comment]
This nerfs the availability for cooking recipes that require those limbs to craft.

**Comment by hugmeorelse** [comment]
> This nerfs the availability for cooking recipes that require those limbs to craft.

They mentioned that as a joke. Not all characters will have a tail anyhow, so that part isn't quite relevant.

**Pull request merged**